tests/models/test_modeling_common.py

```diff
@@ -905,11 +905,13 @@ def test_sharded_checkpoints(self):
             actual_num_shards = len([file for file in os.listdir(tmp_dir) if file.endswith(".safetensors")])
             self.assertTrue(actual_num_shards == expected_num_shards)
 
-            new_model = self.model_class.from_pretrained(tmp_dir)
+            new_model = self.model_class.from_pretrained(tmp_dir).eval()
             new_model = new_model.to(torch_device)
 
             torch.manual_seed(0)
+            _, inputs_dict = self.prepare_init_args_and_inputs_for_common()
```

Comment:
Otherwise the `generator` (which is required as an input when using the `ConsistencyDecoderVAE` will be consumed and its states will change in-place. So, we need to re-initialize here.